### ISSUE TITLE
torch.detach()和.detach_() 

### ISSUE THREAD

**Issue by CHH3213** [CHH3213/Note-Ubuntu_CHH3213#72]
## tensor.detach()

- 返回一个新的tensor，从当前计算图中分离下来的，但是仍指向原变量的存放位置,不同之处只是requires_grad为false，得到的这个tensor永远不需要计算其梯度，不具有grad。
- 即使之后重新将它的requires_grad置为true,它也不会具有梯度grad
- 这样我们就会继续使用这个新的tensor进行计算，后面当我们进行反向传播时，到该调用detach()的tensor就会停止，不能再继续向前进行传播
- 使用detach返回的tensor和原始的tensor共同一个内存，即一个修改另一个也会跟着改变。

## tensor.detach_()
其实detach()和detach_()很像，两个的区别就是detach_()是对本身的更改，detach()则是生成了一个新的tensor
